src/checks/cli.py

```diff
@@ -0,0 +1,40 @@
+import importlib
+import typer
+import logging
+import importlib.metadata
+
+from .requirements import check_packages
+from typing import Optional
+from typing_extensions import Annotated
+
+
+logger = logging.getLogger(__name__)
+
+app = typer.Typer()
+
+def version_callback(value: bool):
+    if value:
+        print(f"checks, version {importlib.metadata.version('checks')}")
+        raise typer.Exit()
+
+
+@app.callback()
+def main(
+    version: Annotated[
+        Optional[bool],
+        typer.Option("--version", callback=version_callback, is_eager=True),
+    ] = None,
+):
+    """
+    A package to centralize data checking functionalities related to SEPLAG's SPLOR subsecretariat.
+    """
+
+
+@app.command("check-packages")
```

Comment:
Não sei se a gente vai ter uso muito uso pra testes via interface CLI, mas se a gente for usar acho que fica mais bonito se a gente chamar somente o nome do teste sem o prefixo (que de certa forma já está no nome do pacote). Ou seja:

```bash
checks installed-packages requirements.in
```

Será que a vale a pena mudar o nome do `[project.scripts]` pra `check` pra ficar

```bash
check installed-packages requirements.in
```